archinstall/lib/installer.py

```diff
@@ -194,9 +194,12 @@ def _verify_service_stop(self) -> None:
 		else:
 			info(tr('Skipping waiting for automatic time sync (this can cause issues if time is out of sync during installation)'))
 
-		info('Waiting for automatic mirror selection (reflector) to complete.')
-		while self._service_state('reflector') not in ('dead', 'failed', 'exited'):
-			time.sleep(1)
+		if not arch_config_handler.args.offline or arch_config_handler.args.skip_mirrors:
+			info('Waiting for automatic mirror selection (reflector) to complete.')
+			while self._service_state('reflector') not in ('dead', 'failed', 'exited'):
+				time.sleep(1)
```

Comment:
Maybe this should terminate at some point to avoid infinite waiting

Reply:
Good idea!